joomla/joomla-cms
# [4.0] Drop for loops

Pull Request for Issue # .

### Summary of Changes
Fix various problems on:
- plg_joomlaupdate
- plg_extensionupdate
- plg_overridecheck

### Testing Instructions
Apply patch and check if the three button work correctly
<img width="571" alt="screenshot 2018-12-28 at 12 11 18" src="https://user-images.githubusercontent.com/3889375/50513699-b5ed1b80-0a99-11e9-9d95-ff2c2d1af09d.png">


### Expected result



### Actual result
Problems: 
- Usage of for loops
- No check before calling a fn on a document element (for whatever reason that element might not exist)
- ESLint


### Documentation Changes Required
No
